docs/source/deployorderer/ordererdeploy.md

```diff
@@ -0,0 +1,299 @@
+
+Before you can launch an orderer in a production network, you need to make sure you've created and organized the necessary certificates, generate the genesis block, decided on storage, and configured `orderer.yaml`.
+
+### Certificates
+
+While **cryptogen** is a convenient utility that can be used to generate certificates for a test environment, it should **never** be used on a production network. The core requirement for certificates for Fabric nodes is that they are Elliptic Curve (EC) certificates. You can use any tool you prefer to issue these certificates (for example, OpenSSL). However, the Fabric CA streamlines the process because it generates the Membership Service Providers (MSPs) for you.
+
+Before you can deploy the orderer, create the recommended folder structure for the orderer or orderer certificates that is described in the [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topic to store the generated certificates and MSPs.
+
+This folder structure isn't mandatory, but these instructions presume you have created it:
+
+```
+├── organizations
+  └── ordererOrganizations
+      └── ordererOrg1.example.com
+          ├── msp
+            ├── cacerts
+            └── tlscacerts
+          ├── orderers
+              └── orderer0.ordererOrg1.example.com
+                  ├── msp
+                  └── tls
+```
+
+You should have already used your certificate authority of choice to generate the orderer enrollment certificate, TLS certificate, private keys, and the MSPs that Fabric must consume. Refer to the [CA deployment guide](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/cadeploy.html) and [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topics for instructions on how to create a Fabric CA and how to generate these certificates. You need to generate the following sets of certificates:
+  - **Orderer organization MSP**
+  - **Orderer TLS CA certificates**
+  - **Orderer local MSP (enrollment certificate and private key of the orderer)**
+
+You will either need to use the Fabric CA client to generate the certificates directly into the recommended folder structure or you will need to copy the generated certificates to their recommended folders after they are generated. Whichever method you choose, most users are ultimately likely to script this process so it can be repeated as needed. A list of the certificates and their locations is provided here for your convenience.
+
+If you are using a containerized solution for running your network (which for obvious reasons is a popular choice), **it is a best practice to mount these folders (volumes) external to the container where the node itself is running. This will allow the certificates to be used to create a new node should the node container go down, become corrupted, or is restarted.**
```

Comment:
My prior comment on this paragraph was not addressed, even though Pam said "Done". Makes me wonder if a whole set of updates was missed in the latest commit?

https://github.com/hyperledger/fabric/pull/2080#discussion_r523108405